src/bundle/Resources/public/js/scripts/admin.limitation.pick.js

```diff
@@ -1,13 +1,162 @@
 (function(global, doc, eZ, React, ReactDOM, Translator) {
-    const udwContainer = doc.getElementById('react-udw');
+    const SELECTOR_LOCATION_LIMITATION_BTN = '.ez-pick-location-limitation-button';
+    const SELECTOR_EZ_TAG = '.ez-tag';
+    const IDS_SEPARATOR = ',';
     const token = doc.querySelector('meta[name="CSRF-Token"]').content;
     const siteaccess = doc.querySelector('meta[name="SiteAccess"]').content;
+    const udwContainer = doc.getElementById('react-udw');
+    const limitationBtns = doc.querySelectorAll(SELECTOR_LOCATION_LIMITATION_BTN);
+    const findLocationsByIdList = (pathArraysWithoutRoot, callback) => {
```

Comment:
Some parts of this file overlap with parts from _src/bundle/Resources/public/js/scripts/udw/select.location.js_, but I wasn't sure if similarities are big enough to make something more generic. What do you think, @sunpietro @dew326 ?